callstats-java-sample/src/io/callstats/sample/CallStatsMainClass.java

```diff
@@ -0,0 +1,68 @@
+package io.callstats.sample;
+
+import io.callstats.sdk.BridgeStatusInfo;
+import io.callstats.sdk.BridgeStatusInfoBuilder;
+import io.callstats.sdk.CallStats;
+import io.callstats.sdk.CallStatsConst;
+import io.callstats.sdk.CallStatsErrors;
+import io.callstats.sdk.CallStatsInitListener;
+import io.callstats.sdk.ServerInfo;
+
+public class CallStatsMainClass {
+	
+	//CallStats Library class
+	private static CallStats callStatslib;
+	
+	private static int appId = 0000000;
+	private static String appSecret = "appSecret";
+	private static String bridgeId = "bridgeId";
+	
+
+	
+	public static void main(String [ ] args)
+	{
+		//EndpointInfo sets the inforamtion realted to the endpoint such as name, os, version etc.
+		ServerInfo endpointInfo = new ServerInfo();
+		endpointInfo.setEndpointType(CallStatsConst.END_POINT_TYPE);
```

Comment:
Minor issue but is there any Java ready made code that can discover endpoint info?